.ci/java-versions.properties

```diff
@@ -4,5 +4,5 @@
 # build and test Elasticsearch for this branch. Valid Java versions
 # are 'java' or 'openjdk' followed by the major release number.
 
-ES_BUILD_JAVA=java10
-ES_RUNTIME_JAVA=java8
+ES_BUILD_JAVA=java11
```

Comment:
@jasontedor I'm actually not sure if this is ok, or we should have it only for matrix and do this lather on.

Reply:
Indeed, we need to hold off on this specific change and leave it to the matrix only for now until we reach conclusion on the minimum runtime JDK for 7.0.0.